src/controllers/article.controller.js

```diff
@@ -150,6 +153,121 @@ class ArticleController {
       return responseGenerator.sendError(res, 500, err.message);
     }
   }
+
+  /**
+   *
+   *
+   * @static
+   * @param {*} request -- Request object
+   * @param {*} response -- Response object
+   * @returns {json} -- Returns a json object
+   * @memberof ArticleController
+   */
+  static async createArticle(request, response) {
+    const { id: authorId } = request.currentUser;
+    const { title } = request.body;
+    const createDate = new Date();
+    try {
+      const article = await BaseRepository.create(db.Article, {
+        ...request.body,
```

Comment:
Destructuring the whole body here might give room for SQL Injection. Please destructure the variables needed here